job-server/test/spark.jobserver/io/JobSqlDAOSpec.scala

```diff
@@ -8,7 +8,7 @@ import com.google.common.io.Files
 import java.io.File
 
 class JobSqlDAOSpec extends TestJarFinder with FunSpecLike with Matchers with BeforeAndAfter {
-  private val config = ConfigFactory.load("local.test.jobsqldao.conf")
+  var config = ConfigFactory.load("local.test.jobsqldao.conf")
```

Comment:
Instead of making this config a var, I think a better practice would be to make a base trait or abstract class called `JobSqlDAOSpecBase`, which defines a config method (`def config: Config`).  Then, have JobSqlDAOSpec extend JobSqlDAOSpecBase and define the config with a val.  Same for JobSqlDAODBCPSpec.  Now you don't need to make the config mutable, which is probably not a good idea.